cli/cmd/pro.go

```diff
@@ -92,10 +103,190 @@ func executeRemoteUpdateToken(ctx context.Context, client *kube.Client, namespac
 	return nil
 }
 
+var offsetsCmd = &cobra.Command{
+	Use:   "update-offsets",
+	Short: "Update Odiglet to use the latest available Go instrumentation offsets",
+	Long: `This command pulls the latest available Go struct and field offsets information from Odigos public server.
+Internet access is required to fetch latest offset manifests.
+It stores this data in a ConfigMap in the Odigos Namespace and updates the Odiglet DaemonSet to mount it.
+
+Use this command when instrumenting apps that depend on very new dependencies that aren't currently supported
+with the previous release of Odigos.
+
+Note that updating offsets does not guarantee instrumentation for libraries with significant changes that
+require an update to Odigos. See docs for more info: https://docs.odigos.io/instrumentations/golang/ebpf#about-go-offsets
+`,
+	Example: `
+# Pull the latest offsets and restart Odiglet
+odigos pro update-offsets
+
+# Revert to using the default offsets data shipped with Odigos
+odigos pro update-offsets --default
+`,
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := cmd.Context()
+		client := cmdcontext.KubeClientFromContextOrExit(ctx)
+		ns := cmd.Flag("namespace").Value.String()
```

Comment:
consider using the existing function [GetOdigosNamespace](https://github.com/odigos-io/odigos/blob/f7408eb106b832a77cfb937a2c538be24a762b1d/cli/cmd/resources/namespace.go#L25)

and remove the namespace from the arguments. this is what we do in all other commands except for install